src/components/input/file/index.tsx

```diff
@@ -21,69 +33,130 @@ export class SmoothlyInputFile implements Clearable, Input {
 	@Prop({ reflect: true }) showLabel = true
 	@Prop({ mutable: true }) value?: File
 	@Prop({ mutable: true, reflect: true }) placeholder: string | undefined
+	@State() dragging = false
 	@Event() smoothlyInputLooks: EventEmitter<(looks: Looks, color: Color) => void>
-	@Event() smoothlyInput: EventEmitter<Record<string, File>>
-	@Event() smoothlyChange: EventEmitter<Record<string, File>>
+	@Event() smoothlyInput: EventEmitter<Record<string, any>>
```

Comment:
should emit smoothly.Data

Reply:
Is it not better to use `Record<string, File | undefined>` since we know that's what it should emit.
Why be less specific, when we know that's what it should emit?

Reply:
The type conforms to `smoothly.Data` and is more specific than `smoothly.Data` so i too think its better. but I can also see that we would want a consistent type on the form. I think its better to have it more specific tho.